test/memory-shared.wast

```diff
@@ -0,0 +1,4 @@
+(module
+ (memory $0 23 256 shared)
+)
+
```

Comment:
I think it's because wasm-dis prints an extra newline, whereas wasm-opt --print does not. Should maybe fix that in another PR.